tv_apps/smart-system/js/homescreen_launcher.js

```diff
@@ -24,6 +25,16 @@
   var HomescreenLauncher = function() {
   };
 
+  HomescreenLauncher.EVENTS = [
```

Comment:
Despite you are right about handleEvent not being called, your statement is not totally true. handleEvent won't be  overwritten by BaseModule, but the other way around. The problem here is that we are overwriting start/stop methods, and baseModule start method is the one that adds the listeners [1]. Using _start/_stop instead should fix the issue. 

Thanks for the review, that was a good catch!

[1] https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/js/base_module.js#L708